include/kernels/dslash_pack.cuh

```diff
@@ -5,10 +5,31 @@
 #include <index_helper.cuh>
 #include <dslash_helper.cuh>
 
+#ifdef NVSHMEM_COMMS
+#include <nvshmem.h>
+#include <nvshmemx.h>
+#include <cuda/atomic>
+
+namespace
+{
+  __device__ cuda::atomic<int, cuda::thread_scope_system> retcount_intra[] {
```

Comment:
doxygen

Reply:
What is wrong with the doxygen in line 14-15?